src/providers/export/testAssetsSplitHelper.ts

```diff
@@ -0,0 +1,31 @@
+import { IAssetMetadata, ITag } from "../../models/applicationState";
+
+/**
+ * A helper function to split train and test assets
+ * @param template String containing variables
+ * @param params Params containing substitution values
+ */
+export function splitTestAsset(allAssets: IAssetMetadata[], tags: ITag[], testSplitRatio: number): string[] {
+    const testAssets: string[] = [];
+
+    if (testSplitRatio > 0 && testSplitRatio <= 1) {
```

Comment:
Consider inverting if statement to reduce code nesting.

Reply:
Fixed in 74a5a81